plugins/labhub.py

```diff
@@ -196,7 +196,7 @@ def unassign_cmd(self, msg, match):
             else:
                 return 'You are not an assignee on the issue.'
 
-    @re_botcmd(pattern=r'mark\s+(wip|pending)\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)/(pull|merge_requests)/(\d+)',  # Ignore LineLengthBear, PyCodeStyleBear
+    @re_botcmd(pattern=r'mark\s+(wip|pending([-\s]*review)?)\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)/(pull|merge_requests)/(\d+)',  # Ignore LineLengthBear, PyCodeStyleBear
```

Comment:
It should be an uncaptured group, else it causes error in command code. Also there space should be there(not optional) between pending and review.

Reply:
Also add tests.